lib/Targets/AIEVecToCpp/TranslateAIEVecToCpp.cpp

```diff
@@ -2520,7 +2590,8 @@ LogicalResult CppEmitter::emitOperation(Operation &op, bool trailingSemicolon) {
                 aievec::SubElemOp, aievec::UPDOp, aievec::UPSOp,
                 aievec::FMAElemOp, aievec::MulElemOp, aievec::BroadcastOp,
                 aievec::BroadcastScalarOp, aievec::MulConvOp, aievec::FMAConvOp,
-                aievec::ShiftOp, aievec::ShuffleOp, aievec::CastOp>(
+                aievec::ShiftOp, aievec::ShuffleOp, aievec::CastOp,
+                aievec::MinOp, aievec::MaxOp>(
```

Comment:
Why are we using custom printers/parsers/verifiers for these ops? Table gen should be able to generate them for us.

Reply:
> We need to translate ops into different intrinsics according to data types. If you want to use table gen only, you can change them in a batch.

That won't prevent us from using auto-generated parsers, printers, and verifiers. It would simplify our code significantly.